deptrac.yaml

```diff
@@ -0,0 +1,139 @@
+deptrac:
+  paths:
+    - ./src
+  layers:
+    - name: Aggregate
+      collectors:
+        - type: directory
+          value: src/Aggregate/.*
+    - name: Attribute
+      collectors:
+        - type: directory
+          value: src/Attribute/.*
+    - name: Clock
+      collectors:
+        - type: directory
+          value: src/Clock/.*
+    - name: Console
+      collectors:
+        - type: directory
+          value: src/Console/.*
+    - name: EventBus
+      collectors:
+        - type: directory
+          value: src/EventBus/.*
+    - name: Lock
+      collectors:
+        - type: directory
+          value: src/Lock/.*
+    - name: Metadata
+      collectors:
+        - type: directory
+          value: src/Metadata/.*
+    - name: Outbox
+      collectors:
+        - type: directory
+          value: src/Outbox/.*
+    - name: Pipeline
+      collectors:
+        - type: directory
+          value: src/Pipeline/.*
+    - name: Projection
+      collectors:
+        - type: directory
+          value: src/Projection/.*
+    - name: Repository
+      collectors:
+        - type: directory
+          value: src/Repository/.*
+    - name: Schema
+      collectors:
+        - type: directory
+          value: src/Schema/.*
+    - name: Serializer
+      collectors:
+        - type: directory
+          value: src/Serializer/.*
+    - name: Snapshot
+      collectors:
+        - type: directory
+          value: src/Snapshot/.*
+    - name: Store
+      collectors:
+        - type: directory
+          value: src/Store/.*
+    - name: WatchServer
+      collectors:
+        - type: directory
+          value: src/WatchServer/.*
+
+  ruleset:
+    Aggregate:
+      - Attribute
+      - Metadata
+    Attribute:
```

Comment:
Right now, we also have a dependency into `Projection` from `Attribute` due to the default values. I dont think that this is a real problem, but if I open this here up, then maybe we could get some problems later on. WDYT?

Reply:
This will be removed soon.